Failure on aarch64-linux [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34815&attempt_id=007e3a28-f3fc-4ca6-8a0a-95c68c4baf02)

<details><summary>Partial log (click to expand)</summary><p>

```
                                                                    ^~~~~
src/sundown/html.c:524:108: warning: unused parameter 'opaque' [-Wunused-parameter]
 toc_link(struct buf *ob, const struct buf *link, const struct buf *title, const struct buf *content, void *opaque)
                                                                                                            ^~~~~~
src/sundown/markdown.c: In function 'htmlblock_end_tag':
src/sundown/markdown.c:1829:22: warning: unused parameter 'rndr' [-Wunused-parameter]
  struct sd_markdown *rndr,
                      ^~~~
builder for '/nix/store/iy2pr2g8bi36lyxpr28nbxripnbvsxd4-ghostwriter-1.5.0.drv' failed with exit code 2
error: build of '/nix/store/iy2pr2g8bi36lyxpr28nbxripnbvsxd4-ghostwriter-1.5.0.drv' failed
```
</p></details>

--- a/pkgs/applications/editors/ghostwriter/default.nix
+++ b/pkgs/applications/editors/ghostwriter/default.nix
@@ -1,0 +1,26 @@
+{ stdenv, fetchFromGitHub, qmake, pkgconfig, qtwebkit, hunspell }:
+
+stdenv.mkDerivation rec {
+  pname = "ghostwriter";
+  version = "1.5.0";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "wereturtle";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0ixw2w2526836lwj4pc0vp7prp1gls7iq37v8m9ql1508b33b9pq";
+  };
+
+  nativeBuildInputs = [ qmake pkgconfig ];
+
+  buildInputs = [ qtwebkit hunspell ];
+
+  meta = with stdenv.lib; {
+    description = "A cross-platform, aesthetic, distraction-free Markdown editor";
+    homepage = src.meta.homepage;
+    license = licenses.gpl3Plus;
+    platforms = platforms.unix;
+    maintainers = with maintainers; [ dotlambda ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -15142,6 +15142,8 @@ with pkgs;
   geany = callPackage ../applications/editors/geany { };
   geany-with-vte = callPackage ../applications/editors/geany/with-vte.nix { };
 
+  ghostwriter = libsForQt5.callPackage ../applications/editors/ghostwriter { };
+
   gksu = callPackage ../applications/misc/gksu { };
 
   gnss-sdr = callPackage ../applications/misc/gnss-sdr { };

